opus/application/static_media/css/opus.css

```diff
@@ -107,13 +107,14 @@ body {
     background-color: white;
 }
 
+/* Note: this is the scrolling area for thumbnail, when min-height is set
```

Comment:
I'm not sure I understand the purpose of this comment. Since min-height is no longer set, will anyone care about this?
If you want to leave it, then:
- Say "thumbnails" and "thumbnails are"
- lower case "we"